src/molecules/formfields/ToggleField/index.js

```diff
@@ -35,13 +35,14 @@ function ToggleField( props ) {
       <div className={styles['button-wrapper']}>
         { toggleChoices.map( (choice, idx) =>
           <Button
-            variant='toggle'
+            variant={input.value === choice.value ? 'toggle-active' : 'toggle'}
             key={`button-toggle-btn-${idx}`}
             className={styles.button}
-            onClick={onClick}
-          >{ choice }</Button>
-          )
-        }
+            onClick={() => input.onChange(choice.value)}
```

Comment:
@jmcolella  yeah sorry my comment was confusing - I meant that the way this is currently implemented, you can pass an `onClick` handler for the buttons, but that the changes I'm proposing would require you to pass `onChange` (and a value) inside of the `input` object to achieve the same functionality which is kinda annoying because we have to behave as if we're redux form in these cases